HeterogeneousCore/AlpakaTest/plugins/alpaka/XyzIdAlgo.h

```diff
@@ -0,0 +1,29 @@
+#ifndef HeterogeneousCore_AlpakaTest_plugins_alpaka_XyzIdAlgo_h
+#define HeterogeneousCore_AlpakaTest_plugins_alpaka_XyzIdAlgo_h
+
+#include <alpaka/alpaka.hpp>
+
+#include "DataFormats/XyzId/interface/alpaka/XyzIdDeviceCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+class XyzIdAlgoKernel {
```

Comment:
As a pattern, would the `XyzIdAlgo.dev.cc` be a better place for the kernel code?

Reply:
I hadn't thought about it, but yes, it should work since `XyzIdAlgoKernel::operator()` should be called only from the wrapper which is itself defined in `XyzIdAlgo.dev.cc`.